README.rst

```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
+learning inference by up to 75%. Currently, Amazon Elastic Inference supports TensorFlow, Apache MXNet, and ONNX
+models, with more frameworks coming soon.
+
+Support for using MXNet with Amazon Elastic Inference in SageMaker is supported in the public `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__.
+
+* For information on how to use the Python SDK to create an endpoint with Amazon Elastic Inference and MXNet in SageMaker, see `Deploying MXNet Models <https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/mxnet#deploying-mxnet-models>`__.
+* For information on how Amazon Elastic Inference works, see `How EI Works <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-how-it-works>`__.
+* For more information in regards to using Amazon Elastic Inference in SageMaker, see `Amazon SageMaker Elastic Inference <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html>`__.
+* For notebook examples on how to use Amazon Elastic Inference with MXNet through the Python SDK in SageMaker, see `EI Sample Notebooks <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-intro-sample-nb>`__.
+
+SageMaker Elastic Inference MXNet container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Amazon Elastic Inference is designed to be used with AWS enhanced versions of TensorFlow serving or Apache MXNet. These enhanced
+versions of the frameworks are automatically built into containers when you use the Amazon SageMaker Python SDK, or you can
+download them as binary files and import them into your own Docker containers. The enhanced MXNet binary are available on Amazon S3, `here <https://s3.console.aws.amazon.com/s3/buckets/amazonei-apachemxnet/>`__.
+
+The `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__ with Amazon Elastic Inference support were built utilizing the
+same instructions listed `above <https://github.com/aws/sagemaker-mxnet-container#building-images>`__ with the
+`CPU Dockerfile <https://github.com/aws/sagemaker-mxnet-container/blob/master/docker/1.3.0/final/Dockerfile.cpu>`__ starting at MXNet version 1.3.0 and above.
+
+The only difference being that the enhanced version of MXNet was passed in for the ``framework_installable`` build-arg.
+
+::
+
+    # Example
+
+    # CPU
+    docker build -t preprod-mxnet:1.3.0-cpu-py3 --build-arg py_version=3
+    --build-arg framework_installable=amazonei_mxnet-1.3.0-py2.py3-none-manylinux1_x86_64.whl -f Dockerfile.cpu .
+
+
+* For information about downloading and installing the enhanced binary for Apache MXNet, see `Install Amazon EI Enabled Apache MXNet <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ei-mxnet.html#ei-apache>`__.
+* For information on which versions of MXNet is supported for Elastic Inference within SageMaker, see `MXNet SageMaker Estimators <https://github.com/aws/sagemaker-python-sdk#mxnet-sagemaker-estimators>`__.
+
+Using MXNet with Amazon Elastic Inference
```

Comment:
this seems like something that belongs in the Python SDK MXNet README

Reply:
The reference to `model_fn` is still in the context of how to use the container on SageMaker, which isn't relevant to this README. If you think it's important to note here that there is code specific to EI in the MXNet image code, then I would suggest adding a new section that talks about the code in this repository, and referencing it there.

Reply:
Fair point. Moving to Python SDK.

Reply:
Created a PR with the doc update for EI in MXNet README. https://github.com/aws/sagemaker-python-sdk/pull/618